public/sites/all/modules/contrib/.gitignore

```diff
@@ -0,0 +1,2 @@
+backup_migrate
+devel
```

Comment:
We should have a `public/sites/all/modules/dev/` folder that setup like the `working` folder and we can include development modules in there